### ISSUE TITLE
Cannot remove the last author once post is saved

### ISSUE THREAD

**Issue by paulschreiber** [Automattic/Co-Authors-Plus#215]
If you assign a co-author to a post, save it, and reload the page, the Remove link disappears. You can no longer remove that author.

**Comment by rebeccahum** [comment]
@paulschreiber  Are you still experiencing this issue?

**Comment by rebeccahum** [comment]
@paulschreiber Are you still experiencing this issue?

**Comment by paulschreiber** [comment]
Yes. The problem still occurs in 3.2.

**Comment by rebeccahum** [comment]
@paulschreiber I'm having trouble reproducing the issue.  Do you have any other plugins that may be affecting CAP?

**Comment by rebeccahum** [comment]
@paulschreiber I'm having trouble reproducing the issue.  Do you have any other plugins that may be affecting Co-Authors Plus?

**Comment by paulschreiber** [comment]
I don't think that's the problem. If you look at the source code, that's what the intended behaviour is:
```
// Hide the delete button if there's only one co-author
```

**Comment by rebeccahum** [comment]
Yes, because you cannot have less than 1 author in a post.

**Comment by paulschreiber** [comment]
No, that's not correct. If you create a post, and your user account isn't associated with a CoAuthor, that post has no authors. You can save or publish that post.

**Comment by rebeccahum** [comment]
I think we are having a little misunderstanding here haha! A co-author,  in the context of a single post, a guest author or user assigned to the post alongside others. Therefore, there needs to be at least one co-author in a post, whether it is a guest author or a user.  On my end of things, I'm still seeing the "Remove" link unless there is one co-author in a post.

**Comment by trepmal** [comment]
Currently, requiring at least one co-author is by design. However, I'm not sure of the original reasoning behind that. I expect it was to mimic core behavior.

@paulschreiber I know your original post was from a while back, but would you happen to have the use case for removing the last author and not replacing it?

**Comment by nielslange** [comment]
Hello @paulschreiber, I assume that you missed @trepmal's last message, as you haven't replied yet.

If I understand your case correctly, you have a regular user (User A) and a guest author (User B). The guest author account is linked to the user account, and you want to be able to show User A as the author of the post instead of User B. Can you confirm that this is what you have in mind?

**Comment by paulschreiber** [comment]
Some posts don't have a byline — like a live blog, or interactive. We wouldn't want to list all 12 people who contributed as authors, as that would be unwiedly.

**Comment by nielslange** [comment]
If I understand you correctly, @paulschreiber, in this case you would have a post, multiple user would write on the same post, but the post itself is not assigned to a single author. Is that correct?

**Comment by paulschreiber** [comment]
The post would not have any CoAuthors.

**Comment by EarthlingDavey** [comment]
Is there any info or a workaround on this?
